src/include/simeng/OS/Process.hh

```diff
@@ -184,8 +184,9 @@ class Process {
    * present, or by calling the `set_tid_address` syscall. */
   uint64_t clearChildTid_ = 0;
 
-  /** The rlimit struct for RLIMIT_STACK. */
-  rlimit stackRlim = {RLIM_INFINITY, RLIM_INFINITY};
+  /** The rlimit struct for RLIMIT_STACK. 0xffffffffffffffff used to represent
+   * RLIM_INFINITY in Linux. */
+  rlimit stackRlim = {0xffffffffffffffff, 0xffffffffffffffff};
```

Comment:
I think it is preferable to define RLIM_INF as a `static constexpr` at the top of this header in case this value needs to be used else where i.e. syscall handler

Reply:
could be moved to the Constants.hh file

Reply:
Added